### ISSUE TITLE
Luminosity contrast ratio of multiple slider controls is less than expected ratio 3:1.

### ISSUE THREAD

**Issue by msftedad** [highcharts/highcharts#24119]
**Test Environment:**
OS: Windows 11 version 25H2 (OS Build 26220.7670
Browsers: Edge (Version 145.0.3800.22)
Tool: Colour Contrast Analyzer Tool
URL: https://www.highcharts.com/demo/stock/flags-general

**User Impact:**
Users with low vision may struggle to identify the slider position because the focus and pointer indicators do not meet the required 3:1 contrast ratio. This makes adjusting values difficult and can lead to inaccurate selections.

**Repro Steps:**

1. Open the URL: https://www.highcharts.com/demo/stock/flags-general in edge browser in windows.
2. Flags marking events page will open.
3. Open Colour Contrast Analyzer Tool.
4. Verify whether Luminosity contrast ratio of slider controls is less than expected ratio 3:1 or not.

**Actual Result:**
Luminosity contrast ratio of multiple controls in less than required ratio 3:1.

- Luminosity ratio of slider icon of graph is '1.6:1', which is less than required '3:1'.
- Luminosity ratio of 'Date Separator' of graph is '1.2:1', which is less than required '3:1'.
- Luminosity ratio of visual indicator of selected area of graph is '2.02:1', which is less than required '3:1'.
- Luminosity ratio of Start and End slider pointers of graph is '1.7:1', which is less than required '3:1'.
- Luminosity ratio of boundary of slider of graph is '2.1:1', which is less than required '3:1'.

**Refer Attachment:**

1. Luminosity contrast ratio of multiple slider controls is less than expected ratio 3:1.png

<img width="1924" height="1071" alt="Image" src="https://github.com/user-attachments/assets/310d4875-7a0f-408a-a138-d541152de388" />

**Expected Result:**
Luminosity contrast ratio of slider controls should be greater than or equals to expected ratio 3:1.

**Comment by kamil-musialowski** [comment]
Hi @msftedad, thank you for reporting that! We will analyze this internally with the Accessibility and design team, however in the meantime you can adjust the colors manually via these options:

(in order of the screenshots)
https://api.highcharts.com/highstock/scrollbar
https://api.highcharts.com/highstock/navigator.handles
https://api.highcharts.com/highstock/navigator.xAxis.gridLineColor
https://api.highcharts.com/highstock/navigator

**Comment by msftedad** [comment]
Hi @kamil-musialowski, Any update?
